lib/galaxy/jobs/__init__.py

```diff
@@ -1402,7 +1402,7 @@ def _collect_metrics( self, has_metrics ):
         job = has_metrics.get_job()
         per_plugin_properties = self.app.job_metrics.collect_properties( job.destination_id, self.job_id, self.working_directory )
         if per_plugin_properties:
-            log.info( "Collecting job metrics for %s" % has_metrics )
+            log.info( "Collecting job metrics for %s: %s" % ( has_metrics, getattr( has_metrics, 'id', None ) ) )
```

Comment:
I think we can also remove the ugly `has_metrics` (i.e. `<galaxy.model.Job object at 0x7fa4dcf903d0>`) from the log, e.g.:
```
            log.info( "Collecting metrics for %s %s" % ( type(has_metrics), getattr( has_metrics, 'id', None ) ) )
```